src/commcare_cloud/ansible/roles/formplayer/tasks/main.yml

```diff
@@ -14,11 +14,21 @@
     user: root
     cron_file: purge_formplayer_files
   with_items:
-    - {name: 'Purge temp files', dir: '/tmp', time_spec: '2d'}
     - {name: 'Purge sqlite db files', dir: '{{ formplayer_data_dir }}', time_spec: '{{ formplayer_purge_time_spec }}'}
   tags:
     - cron
 
+- name: Remove redundant purging cron jobs (cleanup)
```

Comment:
Should this be committed to the code base to be run everytime? Should we just run this ad-hoc once?

Reply:
I think we want to commit it since there are environments outside of our control who may want to run it as well.